etcdutl/snapshot/v3_snapshot.go

```diff
@@ -203,6 +204,16 @@ type RestoreConfig struct {
 	// SkipHashCheck is "true" to ignore snapshot integrity hash value
 	// (required if copied from data directory).
 	SkipHashCheck bool
+
+	// RevisionBump is the amount to increase the latest revision after restore,
+	// to ensure that the revisions never decrease from the clients perspective
```

Comment:
```suggestion
	// to allow administrators to trick clients into thinking that revision never decreased.
```

Reply:
bump revision doesn't ensure anything, it's up to administrators to provide RevisionBump large enough to trick users.